CHANGELOG/CHANGELOG-3.6.md

```diff
@@ -16,6 +16,11 @@ See [code changes](https://github.com/etcd-io/etcd/compare/v3.5.0...v3.6.0).
 ### Deprecations
 
 - Deprecated [V2 discovery](https://etcd.io/docs/v3.5/dev-internal/discovery_protocol/).
+- Removed [etcdctl defrag --data-dir ...](https://github.com/etcd-io/etcd/pull/13793)
```

Comment:
@kkkkun I noticed it now while I was about to merge changes. As you would see the `...` makes it odd/inconsistent with rest of the changelog. Can you please remove it and just keep a period in all four lines added here? It's a nit but also readability related and I don't think we have it in any changelog. So good to keep it consistent. Thank you!

Reply:
Thanks. It would be better than before.